library/ajax/pharmacy_autocomplete/address.txt

```diff
@@ -0,0 +1 @@
+SELECT id, Store_name, address_line_1, city, state FROM pharmacies_weno WHERE Store_name LIKE '%wal%' AND address_line_1 LIKE '%1%' AND city LIKE '%lubbock%'
```

Comment:
What is this?
If it's a script, then needs author/license header, and include globals.php